engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/ValidatorEvaluator.scala

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.engine.definition.validator
```

Comment:
where is it used?

Reply:
NotEmptyValidatorEvaluator extends this trait. All new validators should evaluate based on parameter passed